packages/medusa/src/api/admin/orders/validators.ts

```diff
@@ -79,3 +79,12 @@ export const AdminOrderCreateShipment = z.object({
 export type AdminOrderCreateShipmentType = z.infer<
   typeof AdminOrderCreateShipment
 >
+
+export const AdminOrderCancelFulfillment = z.object({
```

Comment:
The reason I suggested the single Fulfillment Rest API is that I guess this payload will be the same across all separate cancel fulfillment APIs.